workers/data_refinery_workers/processors/smasher.py

```diff
@@ -447,3 +447,1508 @@ def smash(job_id: int, upload=True) -> None:
                         _update_result_objects,
                         _delete_local_files,
                         utils.end_job])
+
+# Look ma, I'm React!
```

Comment:
Can you move this to another file and read in the string?

Reply:
I agree with @kurtwheeler. It will be much easier to review and maintain the content of `BODY_HTML` if it is in a separate html file.

By the way, I copied the HTML into another file and checked it in firefix. The web UI looks pretty simple:
![screenshot from 2018-08-24 12-00-47](https://user-images.githubusercontent.com/15133253/44594920-7c2ba180-a795-11e8-8eb1-1c15c6dd6dde.png)
Why the html code has over 1500 lines with multiple tables and all kinds of CSS styles?

Reply:
Email has been moved to a file.

Email HTML and Web HTML are very different beasts. Different mail services and clients have different constraints. So, all style has to be inlined, the HTML feature set is limited, etc. etc. etc. We use Foundation Email and Zurb Inliner to make this. I was given this as an asset by Ariel/Deepa.